tests/suites/authentication/authentication_test.go

```diff
@@ -0,0 +1,111 @@
+package authentication
+
+import (
```

Comment:
this seems like it could be a kuttl test which might be easier to read and consume